packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -320,6 +329,7 @@ export const getAuthenticationStatusInfo = (account_status: GetAccountStatus) =>
     const poi_resubmit_for_bvi_labuan =
         !poi_pending_for_bvi_labuan && !poi_not_submitted_for_bvi_labuan && !poi_verified_for_bvi_labuan;
     const poi_poa_verified_for_bvi_labuan = poi_verified_for_bvi_labuan && poa_verified;
+    const poa_resubmit_for_labuan = is_authenticated_with_idv_photoid;
```

Comment:
It is the same. Just cerated a variable because any enhancements in future for POA resubmission can be just modified in `getAuthenticationStatusInfo` and the status gets reflected everywhere